src/deploy/functions/services/index.ts

```diff
@@ -117,6 +119,17 @@ const testLabService: Service = {
   unregisterTrigger: noop,
 };
 
+/** A firestore service object */
+const firestoreService: Service = {
+  name: "firestore",
+  api: "firestore.googleapis.com",
+  requiredProjectBindings: noopProjectBindings,
```

Comment:
huh this is a little surprising (that everything would just work w/o additional IAM fixing). Do you expect this to change later?

Reply:
Nope, this integration is a standard Eventarc one, if you look at the other services, only storage requires custom project bindings since they have a __special__ service account